manim/mobject/text/tex_mobject.py

```diff
@@ -358,8 +371,10 @@ def _break_up_by_substrings(self):
         self.submobjects = new_submobjects
         return self
 
-    def get_parts_by_tex(self, tex, substring=True, case_sensitive=True):
-        def test(tex1, tex2):
+    def get_parts_by_tex(
+        self, tex_string: str, substring: bool = True, case_sensitive: bool = True
+    ) -> VGroup:
+        def test(tex1: str, tex2: str) -> bool:
```

Comment:
Maybe a name like `compare_tex` would be better?

Reply:
I like that! I'll make the change.